tasktiger/__init__.py

```diff
@@ -89,8 +89,12 @@ def __init__(self, connection=None, config=None, setup_structlog=False):
             'DEFAULT_QUEUE': 'default',
 
             # After how many seconds time out on listening on the activity
-            # channel and check for scheduled or expired items.
+            # channel and check for scheduled or expired items.  The batch
+            # timeout will delay the specified seconds after the first message
+            # to wait for additional messages, useful for very active systems.
+            # Appropriate values: 0 <= SELECT_BATCH_TIMEOUT <= SELECT_TIMEOUT
             'SELECT_TIMEOUT': 1,
+            'SELECT_BATCH_TIMEOUT': 0,
```

Comment:
Can you elaborate a bit on why this overall helps with high loads? And how is it better than a simple `sleep` in between loop iterations if we haven't processed any tasks in the previous iteration (`not _did_work`)?